rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -27,13 +29,63 @@ local function fetch_request_body()
   return body
 end
 
+-- Returns the certificate for a given host
+function _M.get_cert(host_name)
+  servers_data = configuration_data:get("servers")
+  local ok, servers = pcall(json.decode, servers_data)
+  if not ok then
+    ngx.log(ngx.ERR,  "could not parse servers: " .. tostring(servers))
+    return
+  end
+
+  for _, server in pairs(servers) do
+    if server.Hostname == host_name and server.SSLCertificate then
+      return server.SSLCertificate
+    end
+  end
+
+  ngx.log(ngx.ERR, string.format("Certificate for %s not found", host_name))
+  return 
+end
+
+-- Handler for the /configuration/servers endpoint
```

Comment:
do you think it is hard for someone to figure out that this function is handler for that requests?